### DISCUSSION TITLE
🚀 Introducing Our New Q&A Hub

### DISCUSSION THREAD

**Discussion by theshamkhi** [theshamkhi/GearBox_PC#1] [Q&A]
### Hello Team! 👋

Exciting news! We’ve created a Q&A Discussion space to help us solve problems faster and share knowledge.

- **Ask Questions**: Stuck on something? Don’t hesitate—post it here!
- **Share Solutions**: Help out by answering questions or offering tips.
- **Stay Organized**: Use clear, detailed titles for posts so everyone can find information quickly.

### Happy coding, everyone! 🚀

**Comment by OussamaBenoujja** [comment]
we need to set the html class and  id names for nav and footer the same so we use the same code on the css file or better yet just use a pre made tailwind template @theshamkhi you should add it to each of our teamplates by your own and we will pull it to our local repos.

**Reply by theshamkhi** [reply]
@OussamaBenoujja Yes! I've already used the pre-made Tailwind classes on the **nav** and **footer** tags, because I think those are the only tags we'll use for navigation and the footer throughout the whole project.

**Comment by OussamaBenoujja** [comment]
@theshamkhi Mr Scrum. i want to ask about which platform we should use to host our json data remotely ? i am considiring repl.it as an excellent option.

**Reply by theshamkhi** [reply]
It's an excellent choice but we would rather if you used vercel for its free service
